docs/whats-new/2020-06.md

```diff
@@ -0,0 +1,139 @@
+---
+title: ".NET docs: What's new for June 2020"
+description: "What's new in the .NET docs for June 2020."
+ms.date: 06/29/2020
+---
+
+# .NET docs: What's new for June 2020
+
+Welcome to what's new in the .NET docs for June 2020. This article lists some of the major changes to docs during this period.
+
+## Architecture guides
+
+### Updated articles
+
+- [Work with data](/dotnet/architecture/blazor-for-web-forms-developers/data) - Blazor eBook - Data chapter

```

Comment:
Relative links is possible to be used. It has the advantage that links can work when viewing GitHub too.